cellmlmanip/rdf.py

```diff
@@ -63,10 +63,24 @@ def create_rdf_node(self, node_content=None, fragment_id=None):
 '''
 
 
-def create_rdf_node(namespace_uri, local_name):
-    """Creates and returns an RDF node ``{namespace_uri}local_name``."""
-    # Ensure namespace prefix can be appended to
-    if namespace_uri[-1] not in ['#', '/']:
-        namespace_uri = namespace_uri + '#'
-
-    return rdflib.Namespace(namespace_uri)[local_name]
+def create_rdf_node(node_content):
+    """Creates and returns an RDF node
+
+    :param symbol: node_content EITHER a string or a tuple (namespace_uri, local_name)"""
```

Comment:
Should be `:param node_content: either ...` I think? Perhaps also mention that if a string an RDF literal will be returned. In fact, you could just copy most of the pycml docstring:
```
    Creates an RDF node from the given ``node_content``.

    The ``node_content``, if given, must either be a :class:`rdflib.term.Node`
    instance, a tuple ``(namespace, local_name)``, or a string, in which case
    it is interpreted as a literal RDF node.

    The ``node_content`` may also be `None` to return `None`, allowing easy
    handling of wildcard options to queries.
```

Reply:
That's not necessarily true string starting with # => URIref